base_location/models/res_company.py

```diff
@@ -48,24 +47,22 @@ def _get_company_address_field_names(self):
         return res
 
     def _inverse_city_id(self):
+        self = self.with_context(skip_check_zip=True)
         for company in self:
```

Comment:
Save one line and avoid the overwriting of the variable:
```suggestion
        for company in self.with_context(skip_check_zip=True):
```